docs/usage/shoot_updates.md

```diff
@@ -77,6 +77,9 @@ Apart from the above mentioned triggers, a rolling update of the shoot worker no
 The complete list of fields that trigger a rolling update:
 
 * `.spec.kubernetes.version` (except for patch version changes)
+* `.spec.kubernetes.kubelet.kubeReserved` (unless a worker pool-specific value is set; if feature gate `WorkerPoolHashWithNodeAgentSecret` enabled)
+* `.spec.kubernetes.kubelet.evictionHard` (unless a worker pool-specific value is set; if feature gate `WorkerPoolHashWithNodeAgentSecret` enabled)
+* `.spec.kubernetes.kubelet.cpuManagerPolicy` (unless a worker pool-specific value is set; if feature gate `WorkerPoolHashWithNodeAgentSecret` enabled)
```

Comment:
Do you mind changing the order to `if ...; unless ...`? I find it easier to comprehend this way.